starboard/android/shared/media_codec_bridge_eradicator.h

```diff
@@ -40,14 +40,22 @@ class MediaCodecBridgeEradicator {
 
   bool Destroy(jobject j_media_codec_bridge,
                jobject j_reused_get_output_format_result);
-  void WaitForPendingDestructions();
+  bool WaitForPendingDestructions();
```

Comment:
nit: add comment to say that a return value of false indicates that the wait is timed out.

Reply:
done